TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,60 @@
+﻿#include <iostream>
+
+int searchMax(int array[], size_t len) {
+	int max = array[0];
+	for (int i = 1; i < len; i++)
+	{
+		if (array[i] > max)
+		{
+			max = array[i];
+		}
+	}
+	return max;
+}
+
+int searchMin(int array[], size_t len) {
+	int min = array[0];
+	for (int i = 1; i < len; i++)
+	{
+		if (array[i] < min)
+		{
+			min = array[i];
+		}
+	}
+	return min;
+}
+
+void bubbleSort(int array[], size_t len) {
+	for (int i = 0; i < len - 1; i++) {
+		for (int j = 0; j < len - i - 1; j++) {
+			if (array[j] > array[j + 1]) {
+				int temp = array[j];
+				array[j] = array[j + 1];
+				array[j + 1] = temp;
+			}
+		}
+	}
+}
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	int* array = new int[5];
```

Comment:
One of the worst things I've ever seen. I meant to declare a variable like const int ARRAY_SIZE = argc - 1 and use this down the code.

Reply:
Really misunderstood at first. Fixed.